contracts/test/v0.8/foundry/vrf/VRFV2Plus.t.sol

```diff
@@ -241,6 +249,15 @@ contract VRFV2Plus is BaseTest {
       extraArgs: VRFV2PlusClient._argsToBytes(VRFV2PlusClient.ExtraArgsV1({nativePayment: true}))
     });
     (, uint96 ethBalanceBefore, , ) = s_testCoordinator.getSubscription(subId);
+    vm.expectEmit(true, true, true, true);
+    emit RandomWordsFulfilled(
+      requestId,
+      60034167071501806643072794052071072761405686871992035105939762990515855633675,
```

Comment:
how did we come up with this value? maybe would be helpful to either write a comment or generate this value programmatically by calling `getRandomnessFromProof`. If you decide to do the latter, then you might need to expose `getRandomnessFromProof` using `ExposedVRFCoordinatorV2Plus.sol`

Reply:
Sure, I 

- Exposed getRandomnessFromProof in ExposedVRFCoordinatorV2Plus.sol as getRandomnessFromProofExternal and used it in tests to compute the result
- ExposedVRFCoordinatorV2Plus is only used in tests and is not deployed to any network